### ISSUE TITLE
Clonflict with another addon - SomaFM

### ISSUE THREAD

**Issue by RacerXnyc** [xbmc/visualization.fishbmc#41]
Hey all-

We've been troubleshooting a freeze issue with another addon for the online streaming service SomaFM.  It turns out that the FishBMC visualization was conflicting with it.  Or vise-versa.

I'm not a coder and have no idea where the problem is, and I'm not really looking to get involved further.  BUT- I did want to just drop a note and let you know.  As there may be other conflicts with other music/streaming addons. 

The thread is here-

https://github.com/Soma-FM-Kodi-Add-On/plugin.audio.somafm/issues/12

tl/dr - SomaFM interfaces with the internet Radio group SomaFM (obvs, lol).  On the Nvidia Shield 2015 (the only device tested), when opening a music channel to play, 4 out 5 times the channel will play a split second and then freeze.  1 in 5 times the channel will start to play correctly.  Seems there's a conflict somewhere if the FishBMC visualization  is installed (regardless if enabled or disabled).  Uninstalling FishBMC fixes the problem.

**Comment by AlwinEsch** [comment]
As it is old becomes it closed now, hope for you that the fault no more present after update on https://github.com/Soma-FM-Kodi-Add-On/plugin.audio.somafm

**Comment by RacerXnyc** [comment]
> As it is old becomes it closed now, hope for you that the fault no more present after update on https://github.com/Soma-FM-Kodi-Add-On/plugin.audio.somafm

Hi Alwin,

I just removed the screen visualization, I haven't tested anything further.  Wasn't a big fan of it anyway.  Thanks for the follow up.

**Comment by RacerXnyc** [comment]
It's been working fine for me, thanks.  But, I'm not using 
visualizations anymore, either.


But thanks!


------ Original Message ------
From "Alwin Esch" ***@***.***>
To "xbmc/visualization.fishbmc" 
***@***.***>
Cc "RacerXnyc" ***@***.***>; "Author" 
***@***.***>
Date 5/21/2026 4:44:41 PM
Subject Re: [xbmc/visualization.fishbmc] Clonflict with another addon - 
SomaFM (Issue #41)
